src/VivyFooter/VivyFooter.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Styles from "./vivyFooter.style";
```

Comment:
you could probably just call this file "footer.style.js" even though we have it already. We don't use it in a fashion where we'll get confused imo

Reply:
`styles.js` ?